src/main/java/ch/uzh/ifi/hase/soprafs24/controller/SpotifyController.java

```diff
@@ -1,4 +1,34 @@
 package ch.uzh.ifi.hase.soprafs24.controller;
 
+import ch.uzh.ifi.hase.soprafs24.rest.dto.PlaylistCollectionDTO;
+import ch.uzh.ifi.hase.soprafs24.service.SpotifyService;
+import org.springframework.http.HttpStatus;
+import org.springframework.http.ResponseEntity;
+import org.springframework.web.bind.annotation.GetMapping;
+import org.springframework.web.bind.annotation.RequestMapping;
+import org.springframework.web.bind.annotation.RequestParam;
+import org.springframework.web.bind.annotation.RestController;
+
+@RestController
+@RequestMapping("/user/playlist")
 public class SpotifyController {
+
+    @GetMapping("/names")
+    public ResponseEntity<PlaylistCollectionDTO> getSpotifyUserId(@RequestParam String accessToken) {
```

Comment:
We need to modify this to adhere to our User Centext Security Struct, rather than passing the access token as a request param -> we use the Authorization header